### ISSUE TITLE
com.hazelcast.connector.Hz3* tests fail with Problem in invoking Maven dependency:get OS_TEST_JAR:3.12.13

### ISSUE THREAD

**Issue by sumnerib** [hazelcast/hazelcast#23192]
_master_ (commit 6108cba3d8622fe56fe98b5edb03796c0dc3359e)

Failed on Windows - MicrosoftJDK17: https://jenkins.hazelcast.com/job/Hazelcast-master-Windows-MicrosoftJDK17/41/

The following examples of this issue are from `com.hazelcast.connector.Hz3EnrichmentTest.when_enrichFromInstanceDown_then_shouldThrowJetException`

<details><summary>Example Stacktrace:</summary>

```
java.lang.RuntimeException: Problem in invoking Maven dependency:get OS_TEST_JAR:3.12.13
	at com.hazelcast.test.starter.HazelcastVersionLocator.downloadArtifact(HazelcastVersionLocator.java:96)
	at com.hazelcast.test.starter.HazelcastVersionLocator.locateArtifact(HazelcastVersionLocator.java:81)
	at com.hazelcast.test.starter.HazelcastVersionLocator.locateVersion(HazelcastVersionLocator.java:67)
	at com.hazelcast.test.starter.hz3.Hazelcast3Proxy.newHazelcastInstance(Hazelcast3Proxy.java:61)
	at com.hazelcast.test.starter.hz3.Hazelcast3Starter.newHazelcastInstance(Hazelcast3Starter.java:34)
	at com.hazelcast.connector.BaseHz3Test.setUp(BaseHz3Test.java:107)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:63)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:157)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.io.IOException: Cannot run program "mvn": CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at com.hazelcast.test.starter.HazelcastVersionLocator.downloadArtifact(HazelcastVersionLocator.java:91)
	... 51 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:494)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 53 more

```
</details>
<details><summary>Example Standard output:</summary>

```
BuildInfo right after when_enrichFromInstanceDown_then_shouldThrowJetException(com.hazelcast.connector.Hz3EnrichmentTest): BuildInfo{version='5.3.0-SNAPSHOT', build='20221215', buildNumber=20221215, revision=6108cba, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'when_enrichFromInstanceDown_then_shouldThrowJetException(com.hazelcast.connector.Hz3EnrichmentTest):'
11:15:00, accumulated pauses: 17 ms, max pause: 1 ms, pauses over 1000 ms: 0


No metrics recorded during the test

```
</details>

**Comment by sumnerib** [comment]
Failed again on _master_ with Windows - MicrosoftJDK11: https://jenkins.hazelcast.com/job/Hazelcast-master-Windows-MicrosoftJDK11/43/testReport/junit/com.hazelcast.connector/Hz3EnrichmentTest/testMapUsingReplicatedMap/

**Comment by Patras3** [comment]
Will be fixed in scope of this issue: #23217
